docs/Coiote_IoT_DM/Coiote_videos/How_to_simply_onboard_and_provision_IoT_devices.md

```diff
@@ -0,0 +1,19 @@
+# How to simply onboard and provision IoT devices?
+
+If you've struggled with getting your IoT devices securely connected to the cloud, you're not alone – it's a common challenge for many developers. But here's the good news: we've got a straightforward solution, and using the Coiote IoT Device Management Platform can make it even easier.
+
+To get started, grab a board with Anjay IoT SDK, Zephyr Anjay Client, or any LwM2M Client. You can find the list [here](https://iotdevzone.avsystem.com/docs/LwM2M_Client/Getting_started/).
+
+In our tutorial, we'll guide you through registering your device and ensuring it's securely provisioned.
+We'll cover authentication, confirmed registration, and show you how to check your device's status.
+
+Connect your IoT devices with Anjay IoT SDK - LwM2M Client:
```

Comment:
I'm not sure if the `Anjay IoT SDK - LwM2M Client` is a proper name - for me it's either `Anjay LwM2M Client` or `Anjay IoT SDK`.